test/polydb.jl

```diff
@@ -0,0 +1,167 @@
+        @test Polymake.Polydb.Collection{Polymake.BigObject}(collection_bson) isa Polymake.Polydb.Collection
+        @test Polymake.Polydb.Collection{Polymake.BigObject}(collection_bson) isa Polymake.Polydb.Collection{Polymake.BigObject}
+        constraints = _acp(["DIM" => 3, "N_VERTICES" => 8])
+        query = Dict(constraints...)
+        @test Polymake.Polydb.find(collection_bo, query) isa Polymake.Polydb.Cursor
+        @test Polymake.Polydb.find(collection_bo, query) isa Polymake.Polydb.Cursor{Polymake.BigObject}
+        @test Polymake.Polydb.find(collection_bo, constraints...) isa Polymake.Polydb.Cursor
+        @test Polymake.Polydb.find(collection_bo, constraints...) isa Polymake.Polydb.Cursor{Polymake.BigObject}
+        @test Polymake.Polydb.find(collection_bson, query) isa Polymake.Polydb.Cursor
+        @test Polymake.Polydb.find(collection_bson, query) isa Polymake.Polydb.Cursor{Mongoc.BSON}
+        @test Polymake.Polydb.find(collection_bson, constraints...) isa Polymake.Polydb.Cursor
+        @test Polymake.Polydb.find(collection_bson, constraints...) isa Polymake.Polydb.Cursor{Mongoc.BSON}
+        results_bo = Polymake.Polydb.find(collection_bo, constraints...)
+        @test Polymake.Polydb.Cursor{Mongoc.BSON}(results_bo) isa Polymake.Polydb.Cursor
+        @test Polymake.Polydb.Cursor{Mongoc.BSON}(results_bo) isa Polymake.Polydb.Cursor{Mongoc.BSON}
+        results_bson = Polymake.Polydb.find(collection_bson, constraints...)
+        @test Polymake.Polydb.Cursor{Polymake.BigObject}(results_bson) isa Polymake.Polydb.Cursor
+        @test Polymake.Polydb.Cursor{Polymake.BigObject}(results_bson) isa Polymake.Polydb.Cursor{Polymake.BigObject}
+        @testset "Iterator (Cursor)" begin
+            @test iterate(results_bo) isa Tuple{Polymake.BigObject, Nothing}
+            @test iterate(results_bson) isa Tuple{Mongoc.BSON, Nothing}
+            results_bo = Polymake.Polydb.find(collection_bo, constraints...)
+            results_bson = Polymake.Polydb.find(collection_bson, constraints...)
+            @test collect(results_bo) isa Array{Polymake.BigObject, 1}
+            @test collect(results_bson) isa Array{Mongoc.BSON, 1}
+        end
+        @testset "Iterator (Collection)" begin
+            @test iterate(collection_bo) isa Tuple{Polymake.BigObject, Polymake.Polydb.Cursor{Polymake.BigObject}}
+            @test iterate(collection_bson) isa Tuple{Mongoc.BSON, Mongoc.Cursor}
+            if (get(ENV, "POLYDB_SERVER_URI", "") != "")
+                @test collect(collection_bo) isa Array{Polymake.BigObject}
+                @test collect(collection_bson) isa Array{Mongoc.BSON}
+            end
+        end
+        @testset "Information" begin
+            @test Polymake.Polydb.get_fields(collection_bo) isa Array{String, 1}
+            fields = Polymake.Polydb.get_fields(collection_bo)
+            @test length(fields) == (get(ENV, "POLYDB_SERVER_URI", "") == "" ? 43 : 44)
```

Comment:
```suggestion
            @test length(fields) > 10
```
this should be safe even for the live database, no need to check for the correct number.